build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+output_dir=$1
+
+git_commit_hash=${2:-$(git rev-parse HEAD)}
+
+git_version_string=${3:-$(git log --date=format:%Y%m%d --pretty=~git%cd.%h -n 1)}
+
+build_number=${GITHUB_RUN_NUMBER:=0}
+
+ros_distro=${ROS_DISTRO:=foxy}
```

Comment:
This can be updated to galactic. Otherwise, the build should be run as 
```
ROS_DISTRO=galactic ./build.sh ../.
```

Reply:
Please deliver to develop branch, if you are suggesting changes still to this ros2 update.

Reply:
PR #114 is opened.